d2/src/main/java/com/linkedin/d2/xds/XdsToD2PropertiesAdaptor.java

```diff
@@ -43,6 +43,7 @@
 import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicBoolean;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
unused import

Reply:
Yeah it was there the entire time, my editor just removed it, so I brought it back to avoid a diff in that file. But I can remove it anyway